Libraries/src/Amazon.Lambda.RuntimeSupport/Bootstrap/LambdaBootstrapOptions.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+namespace Amazon.Lambda.RuntimeSupport.Bootstrap
+{
+    public class LambdaBootstrapOptions
+    {
+        public string RuntimeApiEndpoint { get; set; }

```

Comment:
Be sure to add documentation that this should only be set for testing environments. Not when deployed to Lambda.